### PR TITLE
fix: registration-accessibility-submission

### PR DESCRIPTION
**Pull request by ryanbas21**
https://bugster.forgerock.org/jira/browse/SDKS-2696

use svelte key component to recreate the fields that may require focus if no data changes and a submission event occurs.

### CODE REVIEW

**Comment by ryanbas21** [comment]
Your preview environment [pr-156-ryanbas21](https://cloud.okteto.com/#/previews/pr-156-ryanbas21) has been deployed.

  Preview environment endpoint is available [here](https://preview-pr-156-ryanbas21.cloud.okteto.net)

**Comment by cerebrl** [comment]
Can we get some details in the description of this PR? It's currently just blank.

File: src/routes/e2e/widget/inline/+page.svelte [hunk 1]
```diff
@@ -28,9 +29,9 @@
   });
   journeyEvents.subscribe(
     (event: {
-      user: { successful: any; error: any };
-      journey: { error: any };
-      oauth: { error: any };
+      user: { successful: boolean; error: unknown };
+      journey: { error: unknown };
+      oauth: { error: unknown };
```
Comment (cerebrl) [inline src/routes/e2e/widget/inline/+page.svelte:34]: I'd like to remove this type, if we can. It makes it inconsistent with the other e2e test page: https://github.com/ForgeRock/forgerock-web-login-framework/blob/beta/src/routes/e2e/widget/modal/%2Bpage.svelte#L31

**Review by cerebrl — COMMENTED** [review]
Can you make sure to add this `key` wrapper for _all_ callbacks that have policies? You currently just wrap username and password, but there are more. Please review this page and open the "example": https://backstage.forgerock.com/docs/am/7/authentication-guide/authn-supported-callbacks.html. Within the JSON object, look for `policies` and `failedPolicies`. For all of these callbacks, there's a chance that this bug will show up in them too, so let's go ahead and address them all.

**Pull request merged**